src/module/item/deity/data.ts

```diff
@@ -15,7 +15,7 @@ type DeitySystemSource = ItemSystemSource & {
     };
     font: DivineFonts;
     attribute: AttributeString[];
-    skill: SkillAbbreviation | null;
+    skill: SkillAbbreviation[] | null;
```

Comment:
Let's change this to long form while we're at it

Reply:
Changed in latest commit.